pkgs/servers/shellinabox/default.nix

```diff
@@ -1,18 +1,19 @@
-{ stdenv, fetchurl, pam, openssl, openssh, shadow, makeWrapper }:
+{ stdenv, fetchgit, autoconf, automake, libtool, pam, openssl, openssh, shadow, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  version = "2.14";
+  version = "2.16";
   name = "shellinabox-${version}";
 
-  src = fetchurl {
-    url = "https://shellinabox.googlecode.com/files/shellinabox-${version}.tar.gz";
-    sha1 = "9e01f58c68cb53211b83d0f02e676e0d50deb781";
```

Comment:
Use fetchFromGitHub please, fetchgit is non deterministic in most cases with github.